velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -193,10 +193,12 @@ WindowFuzzer::generateSortingKeysAndOrders(
     std::vector<TypePtr>& types) {
   auto keys = generateSortingKeys(prefix, names, types);
   std::vector<SortingKeyAndOrder> results;
-  // TODO: allow randomly generating orders.
   for (auto i = 0; i < keys.size(); ++i) {
-    std::string order = "asc";
-    std::string nullsOrder = "nulls last";
+    auto asc = boost::random::uniform_int_distribution<uint32_t>(0, 1)(rng_);
+    auto order = (asc == 1) ? "asc" : "desc";
```

Comment:
For `order` and `nullsOrder` in SortingKeyAndOrder, let's store velox::core::SortOrder (defined in PlanNode.h) instead of strings to avoid relying on the string literals at multiple places.

Reply:
Updated, thanks for the feedback.